scripts/us_epa/superfund/sites/measurement_sites/generate_measurement_site_mcf.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script generates node mcf files from data/measurement_sites.csv
+"""
+import os
+import re
+from absl import app, flags
+import pandas as pd
+
+DATASET_FILE = 'measurement_sites.csv'
```

Comment:
why not accept this as part of the input / output of generate_mcf? that would make it more flexible for testing too

Reply:
That's a nice suggestion, I have made the change to the script